service/src/main/kotlin/fi/espoo/evaka/shared/dev/DevApi.kt

```diff
@@ -388,6 +389,12 @@ DELETE FROM attachment USING ApplicationsDeleted WHERE application_id = Applicat
         return ResponseEntity.ok().build()
     }
 
+    @DeleteMapping("/employee/all")
```

Comment:
where is this used?

Reply:
It was used at some point. Deleted it now.